### DISCUSSION TITLE
posts/odoo/Odoo-Challenge

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#17] [General]
# posts/odoo/Odoo-Challenge

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/odoo/Odoo-Challenge.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2021-06-02T10:13:48.275Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36 Edg/91.0.864.37",

a = 1

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2022-01-24T19:47:21.966Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36"

// ts = “49146214389324782402070”;
// stmnt = “\"!Odoo - All your applications in one single solution\"“;
// multi = 4730524740;
// ts.slice(eval(“”+’’+””+ƒ(1&lt;0)+””+”-“+’’+””+ƒ(0&lt;1)+”-“+ƒ(1&gt;0))) 为 70
// stmnt.split(“All”).length 为 2
// -(–([,,,undefined].join()).length))[0] = 0 - - - 1 - - - - - 1 - - - - 0 = -2
// pwd.slice(-2) = “57” / “75”
if (parseInt(pwd.slice(-(–([,,,undefined].join()).length))[0]) * parseInt(pwd.slice(0 - - - 1 - - - - - 1 - - - - 0)[1]) * 2 == 70 )
// multi.toString().substr(1,4)+stmnt.substring(2,6) 为 7305Odoo
0===pwd.lastIndexOf(multi.toString().substr(1,4)+stmnt.substring(2,6),0)
// pwd = “7305Odoo57”;

**Reply by dreamhunter2333** [reply]
> 迁移数据
> "createdAt": "2023-01-24T18:03:33.629Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36"

```
if (
  parseInt(pwd.slice(-(–[, , , undefined].join().length))[0]) *
    parseInt(pwd.slice(0 - -(-1) - -(-(-(-1))) - -(-(-0)))[1]) *
    2 ==
  70
```
